docs/conf.py

```diff
@@ -33,43 +34,103 @@
+        # dispatched via XHR without fully reloading the page". See
+        # https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#instant-loading.
+        "navigation.instant"
+
+        # "When sections are enabled, top-level sections are rendered as groups in the sidebar for
+        # viewports above 1220px, but remain as-is on mobile." See
+        # https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-sections.
+        # "navigation.sections"
+
+        # "When tabs are enabled, top-level sections are rendered in a menu layer below the header
+        # for viewports above 1220px, but remain as-is on mobile." See
+        # https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs.
+        # Does not seem to work? This would replace the manually created links in the top bar that
+        # we have in the current documentation.
+        # "navigation.tabs"
+
+        # Above, but keep the tabs visible when scrolling. Doesn't work?
+        # "navigation.tabs.sticky"
+
+        # "Scroll-to-top button". Does not seem to work?
+        # "navigation.top"
+
+        # "When anchor tracking is enabled, the URL in the address bar is automatically updated with
+        # the active anchor as highlighted in the table of contents." See
+        # https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#anchor-tracking.
+        # This is quite neat.
+        "navigation.tracking"
+
+        # Highlight search results on page after clicking a search result. Does not seem to work?
+        # "search.highlight"
+
+        # Share button for a search.
+        "search.share"
+
+        # "When navigation integration for the table of contents is enabled, it is always rendered
+        # as part of the navigation sidebar on the left." Does not seem to work? See
+        # https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-integration.
+        # "toc.integrate",
+    ],
+    "font": False,
+    "palette": [
+        {
+            "media": "(prefers-color-scheme)",
+            "toggle": {
+                "icon": "material/brightness-auto",
+                "name": "Switch to light mode",
+            },
+        },
+        {
+            "media": "(prefers-color-scheme: light)",
+            "scheme": "default",
+            "primary": "white",
+            "accent": "light-blue",
+            "toggle": {
+                "icon": "material/lightbulb-outline",
+                "name": "Switch to dark mode",
+            },
+        },
+        {
+            "media": "(prefers-color-scheme: dark)",
+            "scheme": "slate",
+            "primary": "black",
+            "accent": "light-blue",
+            "toggle": {
+                "icon": "material/lightbulb-outline",
+                "name": "Switch to system preference",
+            },
```

Comment:
To do: decide on colors. I've gone for a slightly simpler "white top bar in light mode" and "black top bar in dark mode" compared to the current docs, but happy to choose some other colors as well if you think that's nicer. The "accent" color still adds a bit of color.

Reply:
Keeping this black and white unless there are objections. I'm also open to changing it later as well.